manifests/dispatch_router.pp

```diff
@@ -30,9 +34,13 @@
   String $logging_level = 'info+',
   Enum['file', 'syslog'] $logging = 'syslog',
   Stdlib::Absolutepath $logging_path = '/var/log/qdrouterd',
+
+  Boolean $service_ensure = false,
 ) {
 
-  contain qpid::router
+  class { 'qpid::router':
+    service_ensure => $service_ensure,
+  }
```

Comment:
Please keep `contain`
```suggestion
  class { 'qpid::router':
    service_ensure => $service_ensure,
  }
  contain qpid::router
```

This ensures correct chaining in the dependency graph. It ensures that all resources from `qpid::router` are contained within this class. It's roughly short form for:
```puppet
include qpid::router
Class['qpid::router'] -> Class['foreman_proxy_content::dispatch_router']
```

Not exactly (because what I just wrote is `require qpid::router`), but I'm not sure how else to explain it.